app/models/manageiq/providers/google/resources.rb

```diff
@@ -0,0 +1,25 @@
+module ManageIQ::Providers::Google
+  class Resources
```

Comment:
How about naming it `Resource`, to me a singular name is more in line with ours and ruby conventions.

And you might make this a real object instead of a helper/parser method:
```ruby
class Resource
  def initialize(uri)    
    # do parsing and set type
  end

  def is_disk?
    # type == 'disks'
  end

  def is_snapshot?
  end

end